constance/admin.py

```diff
@@ -177,6 +177,7 @@ def get_config_value(self, name, options, form, initial):
             'value': localize(value),
             'modified': localize(value) != localize(default),
             'form_field': form[name],
+            'form_field_name': form[name].field.widget.__class__.__name__,
```

Comment:
FWIW, that's not really the "form field name" but the name of the form field's widget class.
In Django this is solved [differently](https://github.com/django/django/blob/6252fd6314f66d2d303cc47c791ffefd27169b42/django/contrib/admin/helpers.py#L138), so let's follow the example:

```python
'is_checkbox': isinstance(form[name].field.widget, forms.CheckboxInput)
```

Let's pass the value to the template instead of implementing it on template level. That also makes sure we don't hard code the class name, while 3rd party theming apps may monkeypaatch the `forms.CheckboxInput` with an own class with a different name.